content/blog/2022/03/2022-03-21-java17-preview-availability.adoc

```diff
@@ -16,7 +16,7 @@ opengraph:
 
 [WARNING]
 --
-Please refer to the link:/doc/administration/requirements/jenkins-on-java-17[Running Jenkins on Java 17] documentation for up-to-date details on how to run Jenkins on Java 17.
+Please refer to the link:/doc/administration/requirements/java[Java requirements] documentation for up-to-date details on how to run Jenkins on Java 17.
```

Comment:
Updating the link in the blog post to ensure that the blog post doesn't contain a broken link.